test/spec/server-discovery-and-monitoring/unified/logging-replicaset.json

```diff
@@ -352,9 +352,6 @@
                 "driverConnectionId": {
                   "$$exists": true
                 },
-                "serverConnectionId": {
```

Comment:
I originally thought that the Node driver does not support serverConnectionId, since it is omitted in Node's cmap logging although it should be required.

Reply:
I've added in serverConnectionId using handshake data. There's also a new item in the Q&A for the kickoff to this ticket concerning a missing TopologyDescriptionEvent emitted by our driver.